src/Campaigns/Models/Campaign.php

```diff
@@ -53,6 +54,7 @@ class Campaign extends Model implements ModelCrud, ModelHasFactory
         'primaryColor' => 'string',
         'secondaryColor' => 'string',
         'goal' => 'int',
+        'goalType' => 'string',
```

Comment:
We should make an enum for this (doesn't have to be now), similar to the donation form [GoalType](https://github.com/impress-org/givewp/blob/162cdcab2d3bbac090efddd1b059085404013091/src/DonationForms/ValueObjects/GoalType.php#L24)

Reply:
Resolved: https://github.com/impress-org/givewp/pull/7547/commits/dae29349803acb504ed17ba09778e647e96bb791